tox.ini

```diff
@@ -0,0 +1,22 @@
+[tox]
```

Comment:
Is `[tox-ansible.ini](https://github.com/kubevirt/kubevirt.core/pull/78/files#diff-56bafa1debb1532b01b6817dcc3bcaa50bd82aa8f85b6e585117c89add5788be)` supposed to be empty?

Reply:
Yes, to use default configuration an empty file is enough. See [here](https://github.com/ansible/tox-ansible?tab=readme-ov-file#usage).